landoui/auth.py

```diff
@@ -0,0 +1,79 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import os
+
+"""Class that governs all authentication with open id connect."""
+from flask_pyoidc.flask_pyoidc import OIDCAuthentication
+
+
+class nullOpenIDConnect(object):
+    """Null object for ensuring test cov if new up fails."""
+
+    def __init__(self):
+        """None based versions of OIDC Object."""
+        pass
+
+
+class OIDCConfig(object):
+    """Convienience Object for returning required vars to flask."""
+    def __init__(self):
+        """General object initializer."""
+        self.OIDC_DOMAIN = os.environ['OIDC_DOMAIN']
+        self.OIDC_CLIENT_ID = os.environ['OIDC_CLIENT_ID']
+        self.OIDC_CLIENT_SECRET = os.environ['OIDC_CLIENT_SECRET']
+        self.LOGIN_URL = "https://{DOMAIN}/login?client={CLIENT_ID}".format(
+            DOMAIN=self.OIDC_DOMAIN,
+            CLIENT_ID=self.OIDC_CLIENT_ID
+        )
+
+    def auth_endpoint(self):
+        return "https://{DOMAIN}/authorize".format(
+            DOMAIN=self.OIDC_DOMAIN
+        )
+
+    def token_endpoint(self):
+        return "https://{DOMAIN}/oauth/token".format(
+            DOMAIN=self.OIDC_DOMAIN
+        )
+
+    def userinfo_endpoint(self):
+        return "https://{DOMAIN}/userinfo".format(
+            DOMAIN=self.OIDC_DOMAIN
+        )
+
+    def client_id(self):
+        return self.OIDC_CLIENT_ID
+
+    def client_secret(self):
+        return self.OIDC_CLIENT_SECRET
+
+
+class OpenIDConnect(object):
+    """Auth object for login, logout, and response validation."""
+
+    def __init__(self, configuration):
+        """Object initializer for auth object."""
```

Comment:
The docstring should describe what `configuration` should be (a dict, some object type).  Check the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html#Comments) for formatting.